frontend/stub/mappings/backend.json

```diff
@@ -71,6 +71,17 @@
         ]
       }
     },
+    {
+      "name": "District Codes",
+      "request": {
+        "url": "/api/clients/districts?page=0&size=250",
```

Comment:
We changed the URL so it's now `/api/districts` instead of `/api/clients/districts`